api/src/main/java/org/pmiops/workbench/notebooks/NotebooksService.java

```diff
@@ -8,13 +10,10 @@
 
 public interface NotebooksService {
 
-  String NOTEBOOKS_WORKSPACE_DIRECTORY = "notebooks";
-  String NOTEBOOK_EXTENSION = ".ipynb";
-
   static String withNotebookExtension(String notebookName) {
```

Comment:
It seems like clone, rename, and copy all rely on withNotebookExtension. It looks like this is only supporting files with a Jupyter extension. Does this need to be adjusted in order to support those functions for rmd files?

Reply:
ah, good catch! We have this to make sure it works with/without file extension. 

- copy - does not pass extension!
- rename - pass the extension
- duplicate - pass the extension
- delete - pass the extension